libraries/AP_DDS/AP_DDS_Client.h

```diff
@@ -187,7 +188,20 @@ class AP_DDS_Client
     };
     static const struct Topic_table topics[];
 
-
+    //! @brief Convenience grouping for a single "channel" of services
+    struct Service_table{
+        //req_id
```

Comment:
What are the purpose of the comments that repeat the variable name?
Feel free to use doxygen comments that explain a bit more like so:
`//! @brief The request ID for the service`